ci-operator/config/openshift/microshift/openshift-microshift-release-4.14.yaml

```diff
@@ -61,6 +75,11 @@ tests:
   commands: make test-unit
   container:
     from: src
+- as: test-rpm
+  cluster: build02
+  commands: make rpm
```

Comment:
Could we do something here to show the output files? Even better would be to somehow verify them. Maybe just listing the contents?

```
for f in $(find _output/rpmbuild/RPMS -type f); do echo; echo $f; rpm -qlp $f; done
```

Reply:
Or maybe

```
for f in $(find _output/rpmbuild/RPMS -type f); do echo; echo $f; rpm -q -p $f --info --dump; done
```

Just to prove that we're building RPMs and they're valid.

Reply:
Added as an additional command here to avoid another commit in microshift. For main branch its embedded in the make-rpm.sh file.